application/frontend/controllers/AuthController.php

```diff
@@ -86,35 +117,40 @@ public function actionLogin()
 
     public function actionLogout()
     {
-        if (\Yii::$app->user->isGuest) {
+        $accessToken = \Yii::$app->request->get('access_token');
+        if ($accessToken !== null) {
```

Comment:
I'm just thinking that if we aren't using sessions for anything, do we want to try to turn off sessions somehow in the PHP settings. If we are using sessions for something (even just some sort of cache), it may still be worth killing the session when they log out.